get.php

```diff
@@ -66,7 +65,6 @@
 	"gl" => "https://github.com/nvdaaddons/nvdaaddons.github.io/wiki/guideLines",
```

Comment:
note to self: how should this be resolved when migrating to the legacy view? append manually?